scripts/actions/spells/trust/lion.lua

```diff
@@ -13,6 +13,21 @@ end
 
 spellObject.onMobSpawn = function(mob)
     xi.trust.message(mob, xi.trust.messageOffset.SPAWN)
+
+    -- Stun all the things!
+    mob:addSimpleGambit(ai.t.TARGET, ai.c.READYING_WS, 0,
+                        ai.r.MA, ai.s.SPECIFIC, 3198) -- Grape Shot (Stun)
```

Comment:
Use a local variable to avoid raw IDs floating around
```lua
local kGrapeShot = 3198

mob:addSimpleGambit(ai.t.TARGET, ai.c.READYING_WS, 0, ai.r.MA, ai.s.SPECIFIC, kGrapeShot)
mob:addSimpleGambit(ai.t.TARGET, ai.c.READYING_MS, 0, ai.r.MA, ai.s.SPECIFIC, kGrapeShot)
-- etc.
```